spec/paperclip/paperclip_spec.rb

```diff
@@ -123,6 +124,10 @@ class ::Four; end
       end
     end
 
+    it 'calls Paperclip::Deprecations.check' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.